classes/label.rb

```diff
@@ -13,4 +14,17 @@ def add_item(item)
     @items << item unless @items.include?(item)
     item.add_label(self)
   end
+
+  private
+
+  def to_s
+    properties = [
+      "id: #{@id}",
+      "name: #{@name}",
```

Comment:
As in the 'constructor' method... where did 'name' come from?

- I would suggest you kindly to remove it.